src/Service/Models/SigninTokenRequest.cs

```diff
@@ -1,6 +1,12 @@
 namespace Passwordless.Service.Models;
 
-public class SigninTokenRequest(string userId) : RequestBase
+public class SigninTokenRequest(string userId, int? timeToLive = null) : RequestBase
 {
     public string UserId { get; } = userId;
+    /// <summary>
+    /// Time to live is the number of seconds the token has before it expires.
+    /// </summary>
+    public int? TimeToLive { get; } = timeToLive;
+
+    public TimeSpan TimeToLiveTimeSpan => new(0, 0, 0, TimeToLive ?? 120);
```

Comment:
This needs to call `FromSeconds`

Reply:
Also, we have the default value of `120` specified now in two different places:

- here
- `MagicLinkTokenRequest`

Reply:
The MagicLinkTokenRequest and SignInTokenRequest are two different requests that do have different. Should be okay to keep them separately defined. Did we want to make them the same length at this point?

Reply:
Sorry, maybe I misunderstood then. I thought they were meant to be the same.

Reply:
@jrmccannon Is correct in that they are separate and I think it makes sense to define them separately.

I do wonder if 15 minutes is too short for an email - not sure?
For MGAT I think at least 15 minutes seems reasonable for a default.